### ISSUE TITLE
An option to go (interactively) ONLY through ambiguous typos 

### ISSUE THREAD

**Issue by yarikoptic** [codespell-project/codespell#2955]
If I run smth like `codespell -i 3 -w` it would go through all typos.  I would like to do something like `codespell -i 3 -w --typos=ambiguous` so only ambiguous typos are checked.  Then I would do a full sweep of `codespell -w` and be done.

**Comment by DimitriPapadopoulos** [comment]
Depending on your definition of ambiguous (multiple suggested fixes?), you can already do that by running codespell twice:
```
codespell -w
codespell -i 2 -w
```

**Comment by yarikoptic** [comment]
Yes, with multiple choices. I want the opposite order of execution - first ambiguous, and then the rest. The reason is that I might need to rerun the automated fix after adding more skips etc
